### ISSUE TITLE
xSQLServerSetup: Get-Resource does not detect following features: BOL, DQ, DQC, SDK, SNAC_SDK

### ISSUE THREAD

**Issue by GoelPooja** [PowerShell/xSQLServer#516]
Get-Resource does not detect following features: BOL, DQ, DQC, SDK, SNAC_SDK.
Error Message: Test-TargetResource returned false after calling set. 

xSQLServerSetup Instance
{
	Action = "Install"
	InstanceName = $SqlInstanceName
	SetupCredential = $DeployServiceAccount
	SourcePath = "C:\Media"
	Features = "FULLTEXT,BC,**DQ**,**BOL**,SSMS,**DQC**,ADV_SSMS,AS,**SDK**,**SNAC_SDK**,SQLENGINE,CONN,RS"
	ProductKey = $ProductKey
}

Plan to fix this by implementing additional Get-Resource method checks for each of these features.

- Operating System: Windows Server 2012 R2
- SQL Server: 2014
- PowerShell: 5
- Module: xSQLServerSetup
- Version: 6.0.0.0

**Comment by johlju** [comment]
If I understood correctly you will send in a PR for this? Then i label this as in progress. 😄 Thanks!

**Comment by GoelPooja** [comment]
Hi Johan,

That's correct. I will send in a PR for this.

Regards,
Pooja Goel

**Comment by johlju** [comment]
@GoelPooja I see SNAC_SDK is listed in this issue. But I can't see that it is added in PR #521. It is not a problem for that PR, you don't have to add it.
It just that this issue will be closed when I merge the PR, so maybe we should open another issue for that? Or did you encounter problem when trying to add it, like it's not possible to add it or there is no point of adding it?

**Comment by GoelPooja** [comment]
Hi Johan,
SNAC_SDK is a default feature for SQL server and there is no obvious way to
detect the feature that is why I did not add it.
I have updated the issue accordingly.
With my understanding no need of this feature in feature list.

On 2 May 2017 12:56 am, "Johan Ljunggren" <notifications@github.com> wrote:

@GoelPooja <https://github.com/GoelPooja> I see SNAC_SDK is listed in this
issue. But I can't see that it is added in PR #521
<https://github.com/PowerShell/xSQLServer/pull/521>. It is not a problem
for that PR, you don't have to add it.
It just that this issue will be closed when I merge the PR, so maybe we
should open another issue for that? Or did you encounter problem when
trying to add it, like it's not possible to add it or there is no point of
adding it?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/PowerShell/xSQLServer/issues/516#issuecomment-298348365>,
or mute the thread
